### ISSUE TITLE
Unify Software Licensing

### ISSUE THREAD

**Issue by kevingy** [SimpleBrowserDotNet/SimpleBrowser#238]
I just noticed that SimpleBrowser and the WebDriver are released under different licenses. SimpleBrowser is a revised BSD license. WebDriver is an Apache license.

It seems like since these projects are part of the same organization that they should be the same license. I'm not an expert to know if that is true or not. It just seems odd.

What do you think, @Teun and @axefrog ?

**Comment by Teun** [comment]
Yeah, that was done before we merged them into an organization. I always
use the Apache license, because it was the first that seemed to fit my
purposes once.

I have no strong opinions. Not sure what the legal side of things is.

Teun

Op di 3 sep. 2019 om 21:57 schreef Kevin Yochum <notifications@github.com>:

**Comment by axefrog** [comment]
Sure, whatever works :+1:
